deepchem/molnet/load_function/tox21_datasets.py

```diff
@@ -37,43 +24,41 @@ def load_tox21(featurizer='ECFP', split='index', reload=True, K=4):
         ' http://deepchem.io.s3-website-us-west-1.amazonaws.com/datasets/tox21.csv.gz'
     )
 
+  tox21_tasks = [
+      'NR-AR', 'NR-AR-LBD', 'NR-AhR', 'NR-Aromatase', 'NR-ER', 'NR-ER-LBD',
+      'NR-PPAR-gamma', 'SR-ARE', 'SR-ATAD5', 'SR-HSE', 'SR-MMP', 'SR-p53'
+  ]
   if featurizer == 'ECFP':
-    featurizer = deepchem.feat.CircularFingerprint(size=1024)
+    featurizer = dc.feat.CircularFingerprint(size=1024)
   elif featurizer == 'GraphConv':
-    featurizer = deepchem.feat.ConvMolFeaturizer()
-  elif featurizer == 'Weave':
-    featurizer = deepchem.feat.WeaveFeaturizer()
+    featurizer = dc.feat.ConvMolFeaturizer()
   elif featurizer == 'Raw':
-    featurizer = deepchem.feat.RawFeaturizer()
+    featurizer = dc.feat.RawFeaturizer()
+  elif featurizer == 'AdjacencyConv':
+    featurizer = dc.feat.AdjacencyFingerprint(max_n_atoms=150, max_valence=6)
 
-  loader = deepchem.data.CSVLoader(
+  loader = dc.data.CSVLoader(
       tasks=tox21_tasks, smiles_field="smiles", featurizer=featurizer)
   dataset = loader.featurize(dataset_file, shard_size=8192)
+  #saved_dataset = copy.deepcopy(dataset)
+  #dataset 
 
   # Initialize transformers 
-  transformers = [
-      deepchem.trans.BalancingTransformer(transform_w=True, dataset=dataset)
-  ]
+  transformers = []
+  #transformers = [
```

Comment:
I'd suggest making this a loading flag to allow users to pick whether or not to do balancing transforms.